projects/packages/forms/src/blocks/contact-form/edit.js

```diff
@@ -89,6 +90,9 @@ const ALLOWED_FORM_BLOCKS = ALLOWED_BLOCKS.concat( CORE_BLOCKS ).filter(
 const PRIORITIZED_INSERTER_BLOCKS = [ ...validFields.map( block => `jetpack/${ block.name }` ) ];
 
 function JetpackContactFormEdit( { name, attributes, setAttributes, clientId, className } ) {
+	// Initialize default form block settings as needed.
+	useFormBlockDefaults( { attributes, setAttributes } );
```

Comment:
We created a new hook that sets/updates the default values for enabling form integrations. 

Initially, the code to set these defaults lived in the integration cards in the modal for each integration. But then the defaults would only be set if a user opened the modal. We want the default applied to all new form blocks, so the defaults need to be set at the form block level. 

I also named this hook generically (useFormBlockDefaults()) because I suspect we'll have other instances where we need to update form block attributes and settings based on PHP-filterable backend code.